src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/ManagedDebuggerImageTypeService.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.Debug
     ///     Provides debuggers with information agbout the target output that the project generates.
     /// </summary>
     [Export(typeof(IDebuggerImageTypeService))]
-    [AppliesTo(ProjectCapability.CSharpOrVisualBasic)]
```

Comment:
re: IIS Express refs. Most of those were stale comments which I have removed. The unit tests happen to leverage IIS Express as an extensibility point, but this code has no knowledge of IIS Express or webs.